README.md

```diff
@@ -185,20 +185,20 @@ Nano ID could be started from number. HTML ID can’t be started from the numbe
 
 ### React Native
 
-To generate secure random IDs in React Native, you must use
-[a native random generator] and asynchronous API:
+To generate secure random IDs in React Native, you must use a native random 
+generator (like [expo-random] or [react-native-securerandom]) and the
+asynchronous API:
 
-```js
-const generateSecureRandom = require('react-native-securerandom').generateSecureRandom
-const format = require('nanoid/async/format')
-const url = require('nanoid/url')
+```ts
+import { getRandomBytesAsync } from 'expo-random';
+import format from 'nanoid/async/format';
+import url from 'nanoid/url';
 
-async function createUser () {
-  user.id = await format(generateSecureRandom, url, 21);
-}
+export const id = () => format(getRandomBytesAsync, url, 21);
```

Comment:
Should we rename `id` to `getId`? But we need to clear that value will be wrapped to Promise. This is why have `createUser` code in the previous example to explain how to use the async code here. Do you have reasons to avoid this example?